vmb/matrimony/models/profiles.py

```diff
@@ -536,6 +533,13 @@ class Expectation(BaseModel):
     ethnicities = models.ManyToManyField(
         Nationality, related_name="ethnicities", blank=True
     )
+    want_NRI = models.CharField(
```

Comment:
This field suits good for only an Indian based context. `NRI` means `Non Resident Indian`. Anyways, at least make the variable name in lowercase.